reference/5.1/Microsoft.PowerShell.Core/About/about_Operators.md

```diff
@@ -521,8 +521,13 @@ You can also create ranges in reverse order.
 ```
 
 The start and end values of the range can be any pair of expressions that
-evaluate to an integer. For example, you could use the members of an
-enumeration for your start and end values.
+evaluate to an integer or a character. The endpoints of the range must be
+convertible to signed 32-bit integers (`[int32]`). Larger values cause an
+error. Also, if the range is captured in an array, the count of resulting is
+limited to maximum size of an array in .NET, which is `[int]::MaxValue - 56`.
```

Comment:
I forgot to mention that this limit only applies to PowerShell (Core) / .NET (Core). In Windows PowerShell / .NET Framework, it is `256MB - 8` (`268435448`): E.g., `(1..268435448).Count` works (if you have the patience), whereas `(1..268435449).Count` fails.